news/2022-09-24-new-featured-artist-ritorikal.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: "New Featured Artist: Ritorikal"
+date: 2022-09-24 10:00:00 +0000
+---
+
+We're proud to welcome **Ritorikal** to our Featured Artist collection!
+
+![](https://assets.ppy.sh/artists/305/header.jpg)
+
+Merging various influences, genres, and his own unending curiosity, producer **Ritorikal** has been generating tunes since as far back as 2009. While electronic music is his shtick, he tends to pull in more organic sounds on top (primarily strings, keys, and guitars), making for a less conventional EDM vibe.
+
+Oh, and **Ritorikal** is also [an osu! player](https://osu.ppy.sh/users/8061786), which miiiight relate to a few of his influences like *MitiS*, *Seven Lions*, [*Koven*](https://osu.ppy.sh/beatmaps/artists/293), and *Virtual Riot*.
```

Comment:
The "which might relate to" doesn't make much sense to me. I don't know what it's trying to say.

Reply:
I've slapped a reword on this to better articulate the (probable) intent of it, at least as far as I understand it.